### ISSUE TITLE
Add a 'break' menu type

### ISSUE THREAD

**Issue by JayBox325** [verbb/navigation#314]
### What are you trying to do?

I have a footer with 4 quick link lists, usually I'd do this with passive items adding a heading where required, but the design doesn't require this. Another option would be to handle this with a single parent to identify a new list then all the child items are the menu items, but this doesn't work for me as it requires understanding of how to handle it and may not be clear to the client.

So, at the moment I've made four "Footer Quicklinks #n" navigation groups. But it would be nicer to have one navigation list that can be split up into as many lists as the front end needs.

### What's your proposed solution?

A way to create groups of separate lists within a single Navigation.

**Comment by engram-design** [comment]
The way we've handled this in in past, for something like the below:

![image](https://user-images.githubusercontent.com/1221575/189504741-ed747e68-b310-42c3-bb28-06427b6dc41d.png)

Is to model it like:
- Services (Passive)
    - Branding (Entry)
    - Web Design (Entry)
    - UI+UX (Entry)
- Agency (Passive)
    - About (Entry)
    - Works (Entry)
    - Career (Entry)

Where the top-level headings don't have a link.

I'd be keen to know what your design looks like, so I can better understand how you're needing to structure nodes?

**Comment by JayBox325** [comment]
Got some strict NDA's in so I've had to comment out anything that has a possible detail in it. However, this is the design, basically mini lists with no headings.

![Screenshot 2022-09-12 at 11 39 32](https://user-images.githubusercontent.com/13233809/189634027-d8242652-a733-454e-a585-b006874071e1.png)

If I did it, I'd do it the same as you but I love the idea of this:

![navigations](https://user-images.githubusercontent.com/13233809/189634743-35319def-66aa-4ef9-b251-e0ee2ffe59ab.jpg)

**Comment by engram-design** [comment]
Thanks for the example (appreciate the NDA), and this could be an interesting approach for sure, and can see the value in that visually. I'd need to give it some thought as how that relates to querying though. If you called `craft.navigation.nodes.all()` on this menu, would you get the first group, or just all?

**Comment by JayBox325** [comment]
it might have to be an intermediate step? `craft.navigation.groups.all()` then there's `group.nodes` within that map. I'm using it all with JSX now so my twig is a bit rusty.

**Comment by engram-design** [comment]
My gut feeling here is that this would probably need to be it's own special type of navigation. Or at least have the "columns" as a setting. I can see where you want to force clients to put things in the right column, but not create more columns freely and break the design.

Would you also want to allow moving items between columns? Visually from your mockup (which I love the look of), it looks like you can't. Architecturally, this would mean X different structures for each group so they are distinct from each other, which is a bit of a change.

Will certainly give it some further thought for down the line. For the moment, you _should_ be able to achieve this with a Passive node to "break" your columns, creating a custom "Break" node type, or using [2-levels](https://github.com/verbb/navigation/issues/314#issuecomment-1242817971).

**Comment by leevigraham** [comment]
@JayBox325 I've been chatting to @engram-design about groups and columns as well.

My solution was to create a custom `Group` type that I targeted in templates. My challenge was including multiple lists in a single column.

In the end we used `display: flex; flex-direction: col;` combined with `flex-basis: 100%` on passive nodes to create full columns.

Current nav build with NEO:

![image](https://user-images.githubusercontent.com/25124/197893433-3c751088-646e-4191-8864-f42318273882.png)

Verbb Navigation Migration:

![image](https://user-images.githubusercontent.com/25124/197893407-84cec396-5166-4ac9-b5c9-4e1cef20c586.png)
